### ISSUE TITLE
Provide some user-friendly indication of what is happening and remove debug statements by default

### ISSUE THREAD

**Issue by stefek99** [decentraland/decentraland-node#49]
I've just wanted to start the app.

![image](https://user-images.githubusercontent.com/249703/27624328-1d9261fe-5bd8-11e7-97fc-3959b068b1d7.png)

I'm waiting and waiting, the only thing that I see is:
```
[info]
[debug]
[debug]
```

I don't know what is really happening - do I have to wait much longer?

Also see the bouncing icon - animated GIF below - what is happening?

![is electron app starting](https://user-images.githubusercontent.com/249703/27624667-a0ee9940-5bd9-11e7-912c-c42e1e3723ef.gif)

---

To provide nice initialisation screen and remove debugging statements by default it would be much more user friendly.

**Comment by stefek99** [comment]
`sh start.sh > stdout.txt 2> stderr.txt`

Some of my logs: https://gist.github.com/stefek99/81645852e59e9dcfb859fe73df57db3e

**Comment by KurzGedanke** [comment]
The bouncing little icons are again `webtorrent-hybrid` which spawns again. Remove the `-hybrid` from the `require('webtorrent-hybrid')` in the `contentdb.js` file. 

And the bronze age is more a proof of concept than a polished finished product, in my opinion the debug logs are very import.

**Comment by ghost** [comment]
+1 to keeping the debug logs for now. It's early stage so developer support is higher priority than user friendliness.

**Comment by stefek99** [comment]
> developer support is higher priority than user friendliness

@berrycoiled agreed

**Comment by martriay** [comment]
+1 @KurzGedanke and @berrycoiled

**Comment by stefek99** [comment]
I'd prefer Quake like press `'` or some magic key...

I didn't know what is happening, what is important, is it still loading...

Furthermore - the Electron torrent app - and dock icon going crazy..

First impression - opened 7 Github issues (one immediately closed as duplicate)

If node by default starts in `mining` mode, then maybe show the hashes, network difficulty, probability, some super basic dashboard, keep it Electron app and maybe the Torrent will not be opening the thing...

And then the magic `'` key to display debugging console.

This is an additional feature, set up a bounty - https://github.com/aragon/aragon-core/labels/bounty - allow members of the community to implement.

**Comment by KurzGedanke** [comment]
The thing is... the electron spawn are a... I think you may call them bugs. Some machines are working better with `webtorrent` other with `webtorrent-hybrid`. So it depends. 

And... the miner show mining information:

```
[info] Miner: hashrate=452khs hashes=8589933 target=503382015 height=130493 best=00000006540e54ed329e9719cd15a91b9765e507e4f4d340c313c0dac6153652
```

**Comment by stefek99** [comment]
> maybe show the hashes, network difficulty, probability, some super basic dashboard


> [info] Miner: hashrate=452khs hashes=8589933 target=503382015 height=130493 best=00000006540e54ed329e9719cd15a91b9765e507e4f4d340c313c0dac6153652


YEAH... Basic chart... I'm just a human, when I see long line full of digits it's not that easy to make sense of it.
